drivers/ethernet/eth_w5500.c

```diff
@@ -56,8 +66,8 @@ static int w5500_spi_read(const struct device *dev, uint32_t addr,
 		.count = 1,
 	};
 	const struct spi_buf rx_buf = {
-		.buf = tmp,
-		.len = ARRAY_SIZE(tmp),
+		.buf = rx_frame_buf,
+		.len = len + 3,
```

Comment:
Creating a new `define`, such as `ETH_PACKET_HEADER_LEN`, for substituting the [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) `3` and `2` throughout this driver would be a welcome change for the readability I believe. To me, it's not clear what the `3` is meant to be, for instance.

Reply:
Added the definitions for them and updated the prefix `MAC` to `ETH`.